src/python/pants/backend/native/tasks/native_task.py

```diff
@@ -72,23 +73,32 @@ def implementation_version(cls):
   def _native_build_settings(self):
     return NativeBuildSettings.global_instance()
 
+  @memoized_property
+  def _native_build_step_settings(self):
+    return NativeBuildStepSettings.global_instance()
```

Comment:
This totally works for now, but I'm planning to move it out afterwards, see https://github.com/pantsbuild/pants/issues/7183#issuecomment-458791268. I think this is the best way to implement this right now so this can stay as-is -- but if you could leave a `TODO(#7183): remove this global subsystem dependency!`, that would be super helpful.